agent/resources/resources_linux.go

```diff
@@ -0,0 +1,119 @@
+// +build linux
+
+// Copyright 2017 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License"). You may
+// not use this file except in compliance with the License. A copy of the
+// License is located at
+//
+//	http://aws.amazon.com/apache2.0/
+//
+// or in the "license" file accompanying this file. This file is distributed
+// on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+// express or implied. See the License for the specific language governing
+// permissions and limitations under the License.
+
+package resources
+
+import (
+	"github.com/aws/amazon-ecs-agent/agent/api"
+	"github.com/aws/amazon-ecs-agent/agent/config"
+	"github.com/aws/amazon-ecs-agent/agent/resources/cgroup"
+	"github.com/cihub/seelog"
+	"github.com/pkg/errors"
+)
+
+// cgroupWrapper implements the Resource interface
+type cgroupWrapper struct {
```

Comment:
Do we expect to add more linux specific resources here in later? In either case, is it reasonable to name this as `cgroupWrapper` or the file as `resources_linux.go`?

Reply:
There is no plan to extend this immediately. 
`cgroupWrapper` just aims to implement the `Resource` interface on `linux`. 
It just wraps up the cgroup object for now!